style.css

```diff
@@ -0,0 +1,552 @@
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.header__social__media,
+.un__icon,
+.nav__items,
+.sponsors,
+.footer {
+  display: none;
+}
+
+.see__program {
+  display: none;
+}
+
+.hamburger.active span:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active span:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active span:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.mobile__nav__menu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  position: fixed;
+  left: -100%;
+  top: 30px;
+  width: 100%;
+  height: 100%;
+  transition: 0.3s;
+  background-color: rgba(0, 0, 0, 0.3);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+}
+
+.mobile__nav__menu.active {
+  left: 0;
+}
+
+.mobile__nav__items {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-left: 20px;
+  gap: 25px;
+  top: 50px;
+  height: 100vh;
+}
+
+.mobile__nav__links {
+  font-weight: 600;
+  color: #fff;
+  font-size: 30px;
+}
+
+.bar {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px;
+  background-color: #272a31;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.home {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./images/bkgrnd.png");
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px;
+  margin-bottom: 30px;
+}
+
+.home__title {
+  font-weight: 300;
+  font-size: 1.5rem;
+  color: #ec5242;
+}
+
+.home__subtitle {
+  font-size: 50px;
+  font-weight: 700;
+  background: url(./images/font-color.jpg);
+  background-clip: content-box;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.home__description {
+  background-color: #d3d3d3;
+  border: 4px solid white;
+  color: #272a31;
+  padding: 1%;
+  width: 60%;
+  margin-top: 20px;
+  margin-right: 100px;
+}
+
+.date {
+  font-size: 1.25rem;
+  font-weight: 900;
+  color: #000;
+  margin-top: 2%;
+}
+
+.address {
+  font-size: 1.25rem;
+  font-weight: 500;
+  color: #000;
+}
+
+.main__program {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/gray-texture.png);
+  height: 100%;
+}
+
+.main__program__title {
+  color: #fff;
+  align-self: center;
+  margin-top: 3%;
+}
+
+.short__hr {
+  align-self: center;
+  border: 1px solid #ef513c;
+  margin-top: 8px;
+  width: 40px;
+}
+
+.program {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-size: 50px;
+  gap: 5px;
+  padding: 12px;
+  margin: 2%;
+}
+
+.program__detailes {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: rgba(61, 64, 71, 0.651);
+  transition: all 0.3s ease-in-out;
+  cursor: pointer;
+  padding: 2%;
+  line-height: 4vh;
+  gap: 20px;
+}
```

Comment:
- [OPTIONAL] KIndly make sure to align the program card contents correctly in order to match the provided Behance design as much as possible.

    - ### YOUR DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/225731253-4ac8eb8e-d94e-4bc0-8135-376f9bd94e05.png)

    - ### BEHANCE DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/225731389-e31ba4a3-d077-429d-b748-85875f5b97dc.png)

Reply:
Will do that, thank you so much.